apps/omg_watcher/test/web/controllers/transaction_test.exs

```diff
@@ -111,6 +113,174 @@ defmodule OMG.Watcher.Web.Controller.TransactionTest do
     end
   end
 
+  #  TODO Remove below skip tags after `OMG-245 - Run tests on (real) postgres database ` .
+  #  There are problems between Ecto and SQlite
+  describe "Controller.TransactionTest - transactions" do
+    @tag :skip
+    @tag fixtures: [:alice, :bob, :phoenix_ecto_sandbox]
+    test "endpoint returns tx that contain requested address as the sender and not recipient", %{
+      alice: alice,
+      bob: bob,
+      phoenix_ecto_sandbox: phoenix_ecto_sandbox
```

Comment:
This `phoenix_ecto_sandbox: phoenix_ecto_sandbox` isn't used - pls remove from all tests